cmd/kpod/version.go

```
@@ -0,0 +1,75 @@
+package main
+
+import (
+	"fmt"
+	"os"
+	"strings"
+	"os/exec"
+	"text/template"
+	"github.com/urfave/cli"
+	"runtime"
+)
+
+//Function to get info for version command
+func getVersion() {
+
+	//Template for the version
+	var versionTemplate = `Client:
+ Version:	{{.Version}}
+ Go Version:	{{.GoVersion}}
+ Git Commit:	{{.GitCommit}}/{{.Version}}
+ Built:		{{.BuildInfo}}
+ OS/Arch:	{{.Os}}/{{.Arch -}}
+ `
+
+ 	var (
+		cmdOut []byte
+		err    error
+	)
+
+	type versionInfo struct {
+	 	Version, GoVersion, GitCommit, BuildInfo, Os, Arch string
+	}
+
+	var info = versionInfo {}
+
+	info.Version = VERSION
+	info.GoVersion = runtime.Version()
+
+	//Using shell commands to get git commit and build info
+	cmdName := "git"
+	cmdArgs := []string{"rev-parse", "--verify", "HEAD"}
+	cmdOut, err = exec.Command(cmdName, cmdArgs...).Output()
+	if err != nil {
+		info.GitCommit = "error: " + err.Error()
```

You should just return the err.